### DISCUSSION TITLE
V3 Enable only Dark mode without Light Mode - is it possible?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#8454] [Help] [answered]
Filament Appearance allow `'dark_mode' => true,` config option what if I prefer to use only dark mode without light mode?

I expect something like this:

`'theme_mode' => 'light' or 'dark',`

**Comment by leandrocfe** [comment, marked as answer]
`$panel->darkMode(true, true)`

**Reply by devhoussam1998** [reply]
thank you @leandrocfe
